tests/comfy_cli/command/test_workflow_edit_nested_autogrow.py

```
@@ -0,0 +1,457 @@
+    next call would refuse."""
+    wf = _minimax_ui_workflow()
+    wf, v = _add_loader(wf, graph, "LoadVideo")
+    for _ in range(3):
+        wf, _ = workflow_ops.connect(wf, graph, v, "VIDEO", 4, "model.reference_videos")
+    with pytest.raises(ValueError) as e:
+        workflow_ops.connect(wf, graph, v, "VIDEO", 4, "model.reference_videos.video_9")
+    assert "at most 3" in str(e.value)
+    assert "next free key" not in str(e.value)
+
+
+def test_a_real_widget_name_outranks_the_bare_element_guess(graph):
```

Pylint (4.0.7)

[warning] Redefining name 'graph' from outer scope (line 50)

(W0621)